models/tokens/optimism/tokens_optimism_erc20_generated.sql

```diff
@@ -0,0 +1,56 @@
+{{ config( alias='erc20_generated', tags=['static'],
```

Comment:
seems this one shouldn't be tagged static?

Reply:
ah yeah copy and paste error probably